webapp/src/css/inbox.less

```diff
@@ -1452,29 +1470,37 @@ mm-sidebar-menu .mat-sidenav-container {
         flex-direction: row;
         align-items: center;
         justify-content: center;
-        .button-label {
-          top: 0;
-          font-size: @font-XXS;
-          padding-top: 5px;
-          text-overflow: ellipsis;
-          margin-top: 0;
+    
+        a .button-label {
+            top: 0;
+            font-size: @font-XXS;
+            padding-top: 5px;
+            margin-top: 3px;
+            overflow: hidden;
+            word-wrap: break-word;
+            color: @nav-icon-gray;
+            text-overflow: ellipsis;
+            white-space: unset;
+            word-break: unset;
+            -webkit-line-clamp: unset;
+            -webkit-box-orient: unset;
+            line-clamp: unset;
```

Comment:
Hi @GallyTi I'm curious whether we need these? 
These styles are causing misalignment of the labels. 

<img width="889" alt="Screenshot 2024-10-28 at 19 19 33" src="https://github.com/user-attachments/assets/1cb92213-3085-4f07-a653-26cad13bd0d4">

Reply:
As you said, they are not needed, when i used the first implementation, it was needed because there was applying the second row for text which we didn't want to have. It is now solved I hope.